app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -52,7 +53,7 @@ export default class SingleAddon extends React.Component{
                 <div className="status-icon" id="stopped-status" />
             }
             {app.appDetails.started === true || app.appDetails.started === false ?
-              app.appDetails && app.appDetails.name
+              <span className="module-click-cursor" onClick={() => handleUserClick(app.appDetails.name)}> {app.appDetails && app.appDetails.name} </span>
```

Comment:
This line is too long, consider using a different format. ie, have the attributes in a new line each. 
Otherwise, good job!